lib/mustache_context.dart

```diff
@@ -124,13 +124,22 @@ class _MustacheContext extends MustacheToString implements MustacheContext {
   }
 
   dynamic _getActualValue(String key) {
-    try {
-      return ctx[key];
-    } catch (NoSuchMethodError) {
-      //Try to make dart2js understand that when we define USE_MIRRORS = false
-      //we do not want to use any reflector
-      return (useMirrors && USE_MIRRORS) ? ctxReflector[key] : null;
+    if (reflect(ctx).type.instanceMembers.containsKey(new Symbol("[]"))) {
```

Comment:
Sorry, was reviewing wrong commit :). It looks good now. I'll try to bake a new version tonight.